wordpress-mobile/AztecEditor-iOS
# Nextpage support

### Details:
In this PR we're:

- Disabling smartDashes: iOS 11 UITextView feature that converts `--` into a single, longer dash, and miserably breaks our HTML editor.
- Renamed MoreAttachmentRenderer > WordPressAttachmentRenderer
- Added support to render the `nextpage` comment, the same way we render the `more` comment

Fixes #742
Needs Review: @diegoreymendez 

### To test:
1. Launch the empty editor
2. Verify that inserting the `more` attachment works as expected
3. Switch to HTML mode
4. Type ``
5. Switch to visual mode and verify everything looks as expected
